### PR TITLE
Vale: Edits for konnect

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary
Running Vale on the `konnect` branch and applying edits based on results.

I've disabled Vale on all topics under `/konnect/legacy`, since those are archived docs that we plan on removing once all accounts are migrated to the new environment. 

### Reason
Preparing branch for eventual release.

### Testing
Tested with Vale locally.

There are still a few Vale errors left that I can't resolve directly. Most of them should be resolved with https://github.com/Kong/docs.konghq.com/pull/4062.

### CODE REVIEW

File: app/konnect/runtime-manager/runtime-groups/index.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ You can sort the table by the `Last Ping` column.
 Column | Description
 -------|-------------
 Host | The hostname of the instance.
-Type | The runtime instance type. Currently, only Kong Gateway is supported.
-Last Ping | The last time that this instance received a configuration update from the Konnect control plane. was used to proxy a Service.
-Version | The Kong Gateway version that this instance is running.
+Type | The runtime instance type. Currently, only {{site.base_gateway}} is supported.
+Last Ping | The last time that this instance received a configuration update from the {{site.konnect_short_name}} control plane. was used to proxy a Service.
```
Comment (cloudjumpercat) [inline app/konnect/runtime-manager/runtime-groups/index.md:128]:
```suggestion
Last Ping | The last time that this instance received a configuration update from the {{site.konnect_short_name}} control plane to proxy a service.
```
There was a partial sentence at the end, so I took my best guess as to what it was supposed to be.

File: app/konnect/servicehub/index.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ gPRC endpoint, GraphQL endpoint, and others.
 * **Service implementation**: The concrete, runnable incarnation of a Service
 version. Each Service version can only have one implementation.
 
-A Konnect service isn't associated with any specific runtime group, but every
+A {{site.konnect_short_name}} service isn't associated with any specific runtime group, but every
 Service version is. When you create a version of the Service, you must select a
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/index.md:38]:
```suggestion
service version is. When you create a version of the service, you must select a
```

File: app/konnect/servicehub/index.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ specifying its protocol, host, port, and path individually.
 
 [Get started with Service management &rarr;](/konnect/servicehub/manage-services)
 
-### Kong Gateway implementations
+### {{site.base_gateway}} implementations
 
 When configuring a {{site.base_gateway}} implementation of a Service, you'll
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/index.md:53]:
```suggestion
When configuring a {{site.base_gateway}} implementation of a service, you'll
```

File: app/konnect/servicehub/index.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ specifying its protocol, host, port, and path individually.
...
 need to specify a Route. Routes determine how (and if) requests get sent to
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/index.md:54]:
```suggestion
need to specify a route. Routes determine how (and if) requests get sent to
```

File: app/konnect/servicehub/index.md [hunk 5]
```diff
@@ -78,7 +78,7 @@ Applies to a specific Service version.
 
 [Upload and publish Service documentation to the Dev Portal &rarr;](/konnect/servicehub/service-documentation)
 
-## Kong Gateway plugins
+## {{site.base_gateway}} plugins
 
 Plugins can be configured to run in a variety of contexts,
 ranging from a specific Service version or Route to all Service versions. Plugins
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/index.md:84]:
```suggestion
ranging from a specific service version or route to all service versions. Plugins
```

File: app/konnect/servicehub/manage-services.md [hunk 6]
```diff
@@ -4,10 +4,10 @@ no_version: true
 ---
 
 Through the [Service Hub](https://cloud.konghq.com/servicehub/), you can
-create and manage all Konnect Services, Service versions, and Service
+create and manage all {{site.konnect_short_name}} Services, Service versions, and Service
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/manage-services.md:7]:
```suggestion
create and manage all {{site.konnect_short_name}} services, service versions, and service
```

File: app/konnect/servicehub/manage-services.md [hunk 7]
```diff
@@ -4,10 +4,10 @@ no_version: true
...
 implementations in one place.
 
-Access all Konnect Service configuration through the {% konnect_icon servicehub %}
+Access all {{site.konnect_short_name}} Service configuration through the {% konnect_icon servicehub %}
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/manage-services.md:10]:
```suggestion
Access all {{site.konnect_short_name}} service configuration through the {% konnect_icon servicehub %}
```

File: app/konnect/servicehub/plugins.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ content_type: reference
 Any {{site.base_gateways}} plugins supported in a self-managed hybrid mode
 deployment are also accessible through Service Hub or the Runtime Manager.
 
-## Kong plugins in Konnect Cloud
+## Kong plugins in {{site.konnect_short_name}}
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/plugins.md:10]:
```suggestion
## Kong plugins in {{site.konnect_saas}}
```

File: app/konnect/servicehub/service-implementations.md [hunk 9]
```diff
@@ -3,10 +3,10 @@ title: Manage Services through ServiceHub
 no_version: true
 ---
 
-An implementation is a Gateway Service. By implementing a Konnect Service
+An implementation is a Gateway Service. By implementing a {{site.konnect_short_name}} Service
 version, you create a Gateway Service in the version's runtime group.
 
-## Implement a Service version (Kong Gateway) {#implement-service-version}
+## Implement a Service version ({{site.base_gateway}}) {#implement-service-version}
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-implementations.md:9]:
```suggestion
## Implement a service version ({{site.base_gateway}}) {#implement-service-version}
```

File: app/konnect/servicehub/service-implementations.md [hunk 10]
```diff
@@ -3,10 +3,10 @@ title: Manage Services through ServiceHub
...
 
 Expose the Service version by pointing it to an upstream service and creating
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-implementations.md:11]:
```suggestion
Expose the service version by pointing it to an upstream service and creating
```

File: app/konnect/servicehub/service-implementations.md [hunk 11]
```diff
@@ -3,10 +3,10 @@ title: Manage Services through ServiceHub
...
 a Route for the proxy. Traffic travelling through this proxy Route can use any
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-implementations.md:12]:
```suggestion
a route for the proxy. Traffic traveling through this proxy route can use any
```

File: app/konnect/servicehub/service-versions.md [hunk 12]
```diff
@@ -3,7 +3,7 @@ title: Manage Konnect Service Versions
 no_version: true
 ---
 
-Every Konnect Service version is associated with one [runtime group](/konnect/runtime-manager/runtime-groups/).
+Every {{site.konnect_short_name}} Service version is associated with one [runtime group](/konnect/runtime-manager/runtime-groups/).
```
Comment (cloudjumpercat) [inline app/konnect/servicehub/service-versions.md:6]:
```suggestion
Every {{site.konnect_short_name}} service version is associated with one [runtime group](/konnect/runtime-manager/runtime-groups/).
```

File: app/konnect/getting-started/configure-service.md [hunk 13]
```diff
@@ -4,12 +4,12 @@ no_version: true
 ---
 
 Using the [Service Hub](/konnect/servicehub), you can create, manage, and
-implement Konnect Services. Each Service consists of at least one
+implement {{site.konnect_short_name}} services. Each service consists of at least one
 Service version, and each Service version can have one implementation.
```
Comment (cloudjumpercat) [inline app/konnect/getting-started/configure-service.md:8]:
```suggestion
service version, and each service version can have one implementation.
```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]

**Comment by lena-larionova** [comment]
Re: the capitalization suggestions, we have decided not to capitalize service, route, consumer, developer, or application going forward. This style guideline is in the currently open style PR (https://github.com/Kong/docs.konghq.com/pull/4043).

File: app/konnect/dev-portal/access-and-approval/manage-app-connections.md [hunk 14]
```diff
@@ -71,8 +70,8 @@ Connections that have been previously revoked can also be reapproved by followin
 
 ### Delete an Application's Connection to a Service {#delete-app-connection}
 
-Delete an application's connection to a Service. A request to register the deleted application
-must be made again by a developer. 
+Delete an application's connection to a service. A developer must request to register the deleted application again.
+must be made again by a developer.
```
Comment (cloudjumpercat) [inline app/konnect/dev-portal/access-and-approval/manage-app-connections.md:74]:
```suggestion

```

**Review by cloudjumpercat — CHANGES_REQUESTED** [review]
Just one small change. It's my bad for leaving it out of my previous review on accident.

**Review by cloudjumpercat — APPROVED** [review]

**Comment by lena-larionova** [comment]
Merging with some vale errors, they will be fixed after another rebase on main.

**Pull request merged**